src/Eris/Generator/Integer.php

```diff
@@ -20,14 +20,17 @@ function int()
 function pos()
 {
     $mustBePositive = function($n) {
```

Comment:
Cheap comment of the day, at this point we should call the variable `mustBeStrictlyPositive` and `mustBeStrictlyNegative` for the `neg`.